### PR TITLE
A unified contributing experience.

### PR DESCRIPTION
**Pull request by dvc94ch**

### CODE REVIEW

**Comment by japaric** [comment]
@dvc94ch tests are failing. The Cargo.toml has a path dependency (`../riscv`). It needs to be changed into a git or crates.io dependency.

**Comment by japaric** [comment]
> tests are failing

See https://travis-ci.org/rust-embedded/riscv-rt/builds/414916253?utm_source=github_status&utm_medium=notification

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+* Respect that people have differences of opinion and that every design or implementation choice carries a trade-off and numerous costs. There is seldom a right answer.
+* Please keep unstructured critique to a minimum. If you have solid ideas you want to experiment with, make a fork and see how it works.
+* We will exclude you from interaction if you insult, demean or harass anyone. That is not welcome behavior. We interpret the term "harassment" as including the definition in the [Citizen Code of Conduct](http://citizencodeofconduct.org/); if you have any lack of clarity about what might be included in that concept, please read their definition. In particular, we don't tolerate behavior that excludes people in socially marginalized groups.
+* Private harassment is also unacceptable. No matter who you are, if you feel you have been or are being harassed or made uncomfortable by a community member, please contact one of the channel ops or any of the [Cortex-M team][team] immediately. Whether you're a regular contributor or a newcomer, we care about making this community a safe place for you and we've got your back.
```
Comment (danc86) [inline CODE_OF_CONDUCT.md]: Should this maybe be s/Cortex-M/RISCV/ ?

I guess you have copy-pasted this quite a few places by now :-)

Comment (dvc94ch) [reply CODE_OF_CONDUCT.md]: Nice catch!

**Pull request closed without merging**